apps/pom.xml

```diff
@@ -109,6 +109,9 @@
 
 		<!-- Whether the preadj endpoints should be enabled -->
 		<local.preadj.enabled>true</local.preadj.enabled>
+
+		<!-- Whether the old MBI hash query feature should be enabled -->
+		<local.preadj.oldMbiHash.enabled>true</local.preadj.oldMbiHash.enabled>
```

Comment:
does this mean that all of the ITs are going to run in this mode?

Reply:
is that what we want? This is somewhat of a special mode for hash rotation, right?

Reply:
Testing with it on ensures the flag is passed through correctly in a running server.

I have also just added a ClaimDaoIT to verify the new query logic works correctly both with and without the flag enabled in a real database rather than mocks.

Reply:
OK, I understand. It's unfortunate that we cannot bring the server up and down in different modes in the ITs but that is way out of scope. Thanks for adding the additional tests.